DN1/docs/demo.jl

```diff
@@ -0,0 +1,53 @@
+#' # DN1 - Naravni zlepek
```

Comment:
Dodajte prosim svoje ime, priimek in vpisno številko, da bom vedel, komu vpisati oceno :-)